src/test/features/response/routes/how-much-paid-claimant.ts

```diff
@@ -0,0 +1,153 @@
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'citizen')
+      })
+
+      checkAlreadySubmittedGuard(app, method, pagePath)
+
+      context('when response not submitted', () => {
+        it('should return 500 and render error page when cannot retrieve claim', async () => {
+          claimStoreServiceMock.rejectRetrieveClaimByExternalId('HTTP error')
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.serverError.withText('Error'))
+        })
+
+        it('should redirect to response type page when response type is not full admission', async () => {
+          draftStoreServiceMock.resolveFind('response', { response: { type: ResponseType.PART_ADMISSION } })
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.redirect.toLocation(ResponsePaths.responseTypePage
+              .evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })))
+        })
+
+        it('should render page when everything is fine', async () => {
+          draftStoreServiceMock.resolveFind('response')
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+
+          await request(app)
+            .get(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.successful.withText('How much have you paid the claimant?'))
+        })
+      })
+    })
+  })
+
+  describe('on POST', () => {
+    const method = 'post'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkNotDefendantInCaseGuard(app, method, pagePath)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(claimStoreServiceMock.sampleClaimObj.defendantId, 'citizen')
+      })
+
+      checkAlreadySubmittedGuard(app, method, pagePath)
+
+      context('when response not submitted', () => {
+        it('should redirect to response type page when response type is not full admission', async () => {
+          draftStoreServiceMock.resolveFind('response', { response: { type: ResponseType.PART_ADMISSION } })
+          claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+
+          await request(app)
+            .post(pagePath)
+            .set('Cookie', `${cookieName}=ABC`)
+            .expect(res => expect(res).to.be.redirect.toLocation(ResponsePaths.responseTypePage
+              .evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })))
+        })
+
+        context('when form is invalid', () => {
+          it('should return 500 and render error page when cannot retrieve claim', async () => {
```

Comment:
The same here, please add a test simulating failure in reading draft since draft is required to render the page.